src/components/NavigationBar/index.tsx

```diff
@@ -88,6 +91,14 @@ const NavigationBar = () => {
         handleMobileMenuClose();
     };
 
+    const navToLogin = async () => {
+        history.push('/login');
+    };
```

Comment:
is the `history` package required, or can we use `props.history.push`, where the history prop comes from `react-router(-dom)`? It has push & replace etc. already, along with components like `<Link>` and `<Redirect>` that can facilitate route change on-click. I'm not sure if having `history` and using the BrowserHistory wrapper that we already have from `react-router-dom` will cause some unexpected side effect to the browser's history stack. Has this been something that's looked into?

Reply:
This one is fine. I see people use both at the same time. 

like this 
https://stackoverflow.com/questions/42701129/how-to-push-to-history-in-react-router-v4

Reply:
recover yarn.lock as well. Please confirm and merge if no problem :)